libsplinter/src/biome/database/schema.rs

```diff
@@ -0,0 +1,26 @@
+table! {
+    notifications (id) {
+        id -> Int8,
+        payload -> Json,
+        created -> Timestamp,
+        origin -> Text,
+        link -> Text,
+        metadata -> Json,
+        recipients -> Array<Text>,
+    }
+}
+
+table! {
+    user_notifications (notification_id) {
+        notification_id -> Int8,
+        user_id -> Int8,
```

Comment:
user_id should perhaps be a string (set by the auth "gateway").

Reply:
updated